services/maven-central/maven-central.tester.js

```diff
@@ -22,7 +22,7 @@ t.create('latest 0.8 version')
 
 t.create('inexistent artifact')
   .get('/v/inexistent-group-id/inexistent-artifact-id.json')
-  .expectBadge({ label: 'maven-central', message: 'invalid' })
+  .expectBadge({ label: 'maven-central', message: 'not found' })
 
 t.create('connection error')
```

Comment:
You can get rid of the "connection error" test as this case is handled in `BaseService` now